extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php

```diff
@@ -427,6 +427,14 @@ public function execute() {
 
 		if( ( $cond1 || $cond2 ) && empty( $wgDevelEnvironment ) ) {
 			$redirectUrl = WikiFactory::getLocalEnvURL( $this->mCityUrl );
+			$hasAuthCookie = !empty( $_COOKIE[\Wikia\Service\User\Auth\CookieHelper::ACCESS_TOKEN_COOKIE_NAME] );
```

Comment:
Just incase, I'd maybe include a check here for any of the cookies we check for on VCL: `(wikicities_session|wikicitiesUserID|UserName|Token|LoggedOut|access_token)`, which is also similar to what we check for elsewhere in MW, i.e. https://github.com/Wikia/app/blob/da75702/includes/OutputPage.php#L1766-L1773

Reply:
commit: dbe66c04017